templates/components/massive_action/add_task.html.twig

```diff
@@ -34,5 +34,8 @@
     {% include('components/itilobject/timeline/form_task_main_form.html.twig') with {
         'no_form': true,
         'disable_pending_reasons': true,
+        'subitem': subitem,
+        'item': item,
+        'mention_options': mention_options,
```

Comment:
Not needed since the values are inherited anyway, but we plan on disabling context inheritance in the future so it doesn't hurt to fix it here ahead of time.